server.js

```diff
@@ -436,8 +436,9 @@ module.exports.initExpress = function () {
   app.use(require('./middlewares/cors'));
   app.use(require('./middlewares/date'));
   app.use(require('./middlewares/effectiveRequestChanged'));
-  app.use('/pl/oauth2login', require('./pages/authLoginOAuth2/authLoginOAuth2'));
-  app.use('/pl/oauth2callback', require('./pages/authCallbackOAuth2/authCallbackOAuth2'));
+  app.use('/pl/google_login', require('./pages/authLoginGoogle/authLoginGoogle'));
+  app.use('/pl/google_callback', require('./pages/authCallbackGoogle/authCallbackGoogle'));
```

Comment:
This change will require a lot of effort and careful coordination when deploying, and AFAICT will result in zero tangible benefit to us. Unless there's a very good reason for making this change, let's revert this and leave it as is. Ditto for `hasGoogleAuth`.

Reply:
It was in the spirit of resolving tech debt while touching these old files. I don't think the impact is as grave as you're making it out to be, but we can also explore that in a different PR. No sense in making this one harder, I guess.

Plus, like I said in my comments, it uncovered a few things we don't really have in the project yet like deprecating config options or highlighting breaking changes.

Reply:
Yep; we generally avoid those types of changes precisely *because* they're a hassle.

> It was in the spirit of resolving tech debt while touching these old files. I don't think the impact is as grave as you're making it out to be, but we can also explore that in a different PR.

To get very specific: this would require adding an additional redirect URI to each environment's OAuth config in Google's console and updating the redirect URI in our own configs. Certainly not insurmountable or particularly grave, but again, this is toil with no benefit for us. As the person deploying and operating this code, I don't like those kind of changes.